src/main/java/de/fraunhofer/aisec/cpg/graph/edge/PropertyEdgeConverter.java

```diff
@@ -51,7 +51,7 @@
       if (deserializer.containsKey(entry.getKey())) {
         Object deserializedProperty = deserializer.get(entry.getKey()).apply(propertyValue);
         result.put(Properties.valueOf(entry.getKey()), deserializedProperty);
-      } else {
+      } else if (!entry.getKey().equals("sub-graph")) {
```

Comment:
Not sure what this is. But at least a comment explaining the use of this constant would be good.

Reply:
I agree with you. Currently the subgraph annotation is realized as an annotation and not with a property field so we cannot set it. This was just preliminary so that the codyze works for now.